app/frontend/src/cljs/teet/ui/common.cljs

```diff
@@ -592,3 +593,19 @@
         [Paper
          (mapc (r/partial context-menu-item toggle!)
                (remove nil? items))]]]]]))
+
+(defn date-label-component
+  "Shows :label with formatted date from :value with an info icon and :tooltip"
+  [{value :value label :label tooltip :tooltip}]
+  [:div
+   {:data-cy "date-label-component"}
+   [:label {:class (<class common-styles/input-label-style false false)}
+    [typography/Text2Bold label]]
+   [:div
+    {:class (<class common-styles/flex-row-center)}
+    [popper-tooltip tooltip
+     [icons/action-info-outlined
+      {:style {:color :prime}}]]
+    [:div
+     {:style {:padding-left "0.3em"}}]
```

Comment:
Either this div needs to wrap the text or the padding can be given to the text element it self.